Realm.Shared/Realm.cs

```diff
@@ -492,6 +492,24 @@ internal SortDescriptorBuilder CreateSortDescriptorForTable(RealmObject.Metadata
         /// <exception cref="RealmObjectManagedByAnotherRealmException">You can't manage an object with more than one realm</exception>
         public void Manage<T>(T obj, bool update = false) where T : RealmObject
         {
+            // This is not obsoleted because the compiler will always pick it for specific types, generating a bunch of warnings
+            ManageInternal(obj, typeof(T), update);
+        }
+
+        /// <summary>
+        /// This realm will start managing a RealmObject which has been created as a standalone object.
+        /// </summary>
+        /// <param name="obj">Must be a standalone object, null not allowed.</param>
+        /// <param name="update">If true, and an object with the same primary key already exists, performs an update.</param>
+        /// <exception cref="RealmInvalidTransactionException">If you invoke this when there is no write Transaction active on the realm.</exception>
+        /// <exception cref="RealmObjectManagedByAnotherRealmException">You can't manage an object with more than one realm</exception>
+        public void Manage(RealmObject obj, bool update = false)
+        {
+            ManageInternal(obj, obj.GetType(), update);
```

Comment:
If `obj` is null here there will be a `NullReferenceException` before `ManageInternal` throws `ArgumentNullException`.